cmd/minikube/cmd/env.go

```diff
@@ -55,6 +64,19 @@ func buildDockerEnvShellOutput(envMap map[string]string) string {
 	return output
 }
 
+func buildUnsetShellOutput() string {
+	envArr := []string{"DOCKER_TLS_VERIFY", "DOCKER_HOST", "DOCKER_CERT_PATH"}
+	output := ""
+	for _, envName := range envArr {
+		output += fmt.Sprintf("unset %s\n", envName)
```

Comment:
Do we need to support windows & other shells as we do when creating the docker env output? I guess unset doesn't work across all shells?